distributed/scheduler.py

```diff
@@ -3579,6 +3580,16 @@ async def add_worker(
         if address in self.workers:
             raise ValueError("Worker already exists %s" % address)
 
+        if nbytes:
+            err = (
+                f"Worker {address!r} reconnected with {len(nbytes)} keys in memory! Worker reconnection is not supported. "
+                f"{list(nbytes)}"
+            )
+            logger.error(err)
+            if comm:
+                await comm.write({"status": "error", "message": err, "time": time()})
+            return
```

Comment:
Is this likely to occur with workers of this version or later?

Reply:
807c57aeb1eb8839d01fe998d2ce4d3678a8ade8